src/integrations/watchers/auto-update-watcher.php

```diff
@@ -0,0 +1,215 @@
+<?php
+
+namespace Yoast\WP\SEO\Integrations\Watchers;
+
+use Yoast\WP\SEO\Conditionals\No_Conditionals;
+use Yoast\WP\SEO\Helpers\Notification_Helper;
+use Yoast\WP\SEO\Helpers\Product_Helper;
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+use Yoast\WP\SEO\Presenters\Admin\Auto_Update_Notification_Presenter;
+use Yoast_Notification;
+use Yoast_Notification_Center;
+
+/**
+ * Shows a notification for users who have WordPress auto updates enabled but not Yoast SEO auto updates.
+ */
+class Auto_Update_Watcher implements Integration_Interface {
+
+	use No_Conditionals;
+
+	/**
+	 * The notification ID.
+	 */
+	const NOTIFICATION_ID = 'wpseo-auto-update';
+
+	/**
+	 * The Yoast notification center.
+	 *
+	 * @var Yoast_Notification_Center
+	 */
+	protected $notification_center;
+
+	/**
+	 * The notification helper.
+	 *
+	 * @var Notification_Helper
+	 */
+	protected $notification_helper;
+
+	/**
+	 * The product helper.
+	 *
+	 * @var Product_Helper
+	 */
+	protected $product_helper;
+
+	/**
+	 * Auto_Update constructor.
+	 *
+	 * @param Yoast_Notification_Center $notification_center The notification center.
+	 * @param Notification_Helper       $notification_helper The notification helper.
+	 * @param Product_Helper            $product_helper      The product helper.
+	 */
+	public function __construct(
+		Yoast_Notification_Center $notification_center,
+		Notification_Helper $notification_helper,
+		Product_Helper $product_helper
+	) {
+		$this->notification_center = $notification_center;
+		$this->notification_helper = $notification_helper;
+		$this->product_helper      = $product_helper;
```

Comment:
You can remove this dependency because it isn't used.

Reply:
Done